tests/Extensions/Hosting.Services.UnitTests/OmexHostedServiceTests.cs

```diff
@@ -44,24 +40,22 @@ public async Task StartAsync_ProperlyCanceled()
 		[TestMethod]
 		public async Task StartAsync_HandlesExceptions()
 		{
-			MockRunner runnerMock = new MockRunner(t => Task.Run(() => throw new Exception("Totaly valid exeption")));
-			MockLifetime lifetimeMock = new MockLifetime();
+			MockRunner runnerMock = new MockRunner(t => Task.Run(async () =>
+				{
+					await Task.Delay(5);
+					throw new Exception("Totaly valid exeption");
```

Comment:
>Exception [](start = 15, length = 9)

If we add static analysers such as SonarQube to the project, they will complain about throwing a generic `Exception`. Therefore, it might be better to throw something more specific here now

Reply:
Interesting point, changed the type of exception, thank you